app/controllers/api/v1/conversations/messages_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  module V1
+    module Conversations
+      class MessagesController < Api::V1::ApiController
+        helper_method :conversation, :user
+
+        def create
+          @message = conversation.messages
+                                 .create!(message_params)
+          @message.update!(user_id: user.id)
+          @message.notify
```

Comment:
Shouldn't be an after create of message this `notify` method? Or do you sometimes need to create a message without notifying the user?

Reply:
@leandrodzp  I do sometimes need that, since when a target matches, a new message is created (regarding new match), but notification goes for target match instead.